sdk/storagecache/arm-storagecache/src/storageCacheManagementClientContext.ts

```diff
@@ -45,7 +45,7 @@ export class StorageCacheManagementClientContext extends msRestAzure.AzureServic
 
     super(credentials, options);
 
-    this.apiVersion = '2019-11-01';
+    this.apiVersion = '2020-03-01';
```

Comment:
It's better to bump a major version based on the apiversion has changed.

Reply:
The version in Line 16 of this file also needs to be consistent with package.json

Reply:
done